### DISCUSSION TITLE
nsclient calling external python scripts how to

### DISCUSSION THREAD

**Discussion by joaormp** [mickem/nscp#789] [General]
Hi guys,

I have some python scripts working on application monitoring and I need to find the sintax to use on the nsclient.ini

on powershell for instance I use:
[service_name] = cmd /c echo scripts\[name_of_the_script.ps1] $ARG1$ ; exit $LastExitCode | powershell.exe -noprofile -executionpolicy bypass -command -

but for python scripts I'm not able to find the correct syntax.

Does anyone can help on this topic?

**Comment by kragebein** [comment]
The last time the python documentation was updated was in 2017, it just says "todo".
I dont think you'll get a reply soon.

**Comment by mickem** [comment]
What kinb of script is it?

If it is a general script you can just run it like you do other scripts...

https://nsclient.org/docs/howto/external_scripts/

```
[/settings/external]
my_script=python scripts\script.py
```

There is a "PythonScript" module as well which is only used for script interacting *with* NSClient++ i.e. making changes or running commands or some such. For regular script they work just like regular scripts...
